src/components/Media/Video/Video.tsx

```diff
@@ -103,7 +106,11 @@ const Video = (props: VideoAllProps) => {
 
     const defaultVideoBlock = useMemo(() => {
         return video.src.length && !hasVideoFallback ? (
-            <div className={b('wrap', videoClassName)} style={{height}}>
+            <div
+                className={b('wrap', videoClassName)}
+                style={{height}}
+                data-qa={defaultMediaVideoQa}
```

Comment:
I saw in other places `qa || defaultMediaVideoQa`
Should we add here custom qa too?

Reply:
Slightly changed common way we assert qa attributes. Thank you for point on this pattern